packages/storybook/src/generators/init/init.ts

```diff
@@ -210,7 +212,11 @@ function editRootTsConfig(tree: Tree) {
   }
 }
 
-export function initGenerator(tree: Tree, schema: Schema) {
+export async function initGenerator(tree: Tree, schema: Schema) {
+  await jsInitGenerator(tree, {
```

Comment:
I think this should go in the `configuration` generator, rather than here. The `initGenerator` does not generate stories. It can be invoked on it's own and it will not generate stories. Stories are only generated by the `configuration` generator, so the `js` option and `jsinit` should only be there, I believe.

Reply:
the js option here is to determine whether to add typescript in package.json. storybook's init generator mostly deals with dependencies and it does the deps installation, so I think jsInit generator should go there since it also add to deps.

Reply:
Ah I see. But someone may call the init generator and not call the configuration generator at all, so they would not need the `js`... But again, if you think it makes sense here, then ok!